cmd/web-api/main.go

```diff
@@ -1,19 +1,21 @@
 package main
 
 import (
+	"github.com/ucl-arc-tre/portal/internal/config"
 	"github.com/ucl-arc-tre/portal/internal/graceful"
 	handler "github.com/ucl-arc-tre/portal/internal/handler/web"
 	"github.com/ucl-arc-tre/portal/internal/middleware"
 	openapi "github.com/ucl-arc-tre/portal/internal/openapi/web"
+	"github.com/ucl-arc-tre/portal/internal/rbac"
 	"github.com/ucl-arc-tre/portal/internal/router"
 )
 
 func main() {
-	_ = graceful.NewDB()
+	rbac.Init()
```

Comment:
Would it make sense to initialise the DB and RBAC separately, and pass in the DB to the RBAC setup?

Reply:
ah so the `NewDB` call before was just a stub to make sure the API could init the DB. now we have something that needs the db (the RBAC init) it's run in that

Reply:
Makes sense, I was wondering whether it still makes sense to initialise them separately. No strong feelings about it, especially if the RBAC stuff and the DB are closely linked.

Reply:
fair. the RBAC & DB shouldn't be closely linked. I think there's an argument for separating the initialisations, but given the db init is idempotent I'm reasonably happy with `NewDB` initialising if it needs and handing over a connection. happy to change if you feel strongly

Reply:
I would personally separate them. But I'll leave the final decision to you as the current implementation does make sense still.